<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>698</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: #17181d;
	color: #ffffff;
}
QPushButton {
	font: 63 20pt &quot;DejaVu Sans Mono&quot;;
	background-color: #e5e5e5;
	border: 1px solid #e5e5e5;
	border-radius: 5px;
	color: #17181d;
}
QPushButton:hover {
    border-radius:5px;
	border: 1px solid #a0a0a0;
	background-color: #a0a0a0;
    color: 17181d;
}

QPushButton:disabled {
	border-radius:5px;
	border: 1px solid gray;
    background-color: gray;
    color: 17181d;
}

QPushButton:pressed {
    border-radius:5px;
	border: 1px solid #808080;
	background-color: #808080;
    color: 17181d;
}
QHeaderView::section {
 background-color: #303030;
border:none;
 }

QScrollBar:vertical {
        background-color: #333333;
        width: 10px;
    }
    QScrollBar::handle:vertical {
        background-color: #555555;
        min-height: 20px;
    }
    QScrollBar::handle:vertical:hover {
        background-color: #777777;
    }
    QScrollBar::add-line:vertical,
    QScrollBar::sub-line:vertical {
        background: none;
    }
    QScrollBar::add-page:vertical,
    QScrollBar::sub-page:vertical {
        background: none;
    }


QLabel {
    font: 63 20pt &quot;DejaVu Sans Mono&quot;;
    background-color: #292c35;
    border: 1px solid #292c35;
    border-radius: 5px;
}

QListWidget {
    font: 63 16pt &quot;DejaVu Sans Mono&quot;;
    background-color: #292c35;
    border: 1px solid #292c35;
    border-radius: 5px;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>All old usages</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="all_data_list">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_button">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>selected data</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="all_data_list_2">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_button">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
